### DISCUSSION TITLE
Alway print all screen on Ubuntu

### DISCUSSION THREAD

**Discussion by danh996** [flameshot-org/flameshot#3034] [Q&A]
I have install newest Ubuntu and Install Flameshot few day ago. It seem to updated.
At last version, Flameshot alway show select a area and multiple option: draw, pen,.. on this.

![flameshot-hero](https://user-images.githubusercontent.com/74217371/209434820-6745e958-a650-4d53-8ea7-8423027fbb22.jpg)

But in my computer when I print screen it alway print all screen and show a popup like this:
![62d041576771bf2fe660](https://user-images.githubusercontent.com/74217371/209434875-65f8c02c-3f31-41c9-a84b-7f3255e5b14e.jpg)

I installed by this command: `apt install flameshot`

Does it have any updated on Flameshot like this?

Many thanks.

**Comment by mmahmoudian** [comment]
https://flameshot.org/docs/guide/wayland-help/#i-am-asked-to-share-my-screen-every-time
